src/languages/sas.js

```diff
@@ -4,88 +4,492 @@ Author: Mauricio Caceres <mauricio.caceres.bravo@gmail.com>
 Description: Syntax Highlighting for SAS
 */
 
+import * as regex from '../lib/regex.js';
+
+/** @type LanguageFn */
 export default function(hljs) {
   // Data step and PROC SQL statements
-  const SAS_KEYWORDS =
-    'do if then else end until while ' +
-    '' +
```

Comment:
@mcaceresb  Is there a reason for these line breaks?  Are there implicit headings we should add and make explicit?

Reply:
@joshgoebel  I don't think so. I think it was just a visual aid when I was coding.